### PR TITLE
Feature/907 initial manager ux

### PR DESCRIPTION
**Pull request by seanbrookes**
@anthonyettinger

### CODE REVIEW

**Comment by seanbrookes** [comment]
test please

**Comment by anthonyettinger** [comment]
LGTM

**Comment by anthonyettinger** [comment]
test please

File: client/www/scripts/modules/manager/manager.services.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Manager.service('ManagerServices', [
+  '$http',
+  '$log',
+  function($http, $log) {
+    var svc = this;
+
+    svc.getManagerHosts = function() {
+
+      return $http.get('./scripts/modules/manager/test-data.json')
```
Comment (ritch) [inline client/www/scripts/modules/manager/manager.services.js]: Can we remove this now?

File: client/www/scripts/modules/manager/test-data.json [hunk 2]
```diff
@@ -0,0 +1,217 @@
+[
```
Comment (ritch) [inline client/www/scripts/modules/manager/test-data.json]: Is this file still required?

**Comment by seanbrookes** [comment]
test please

**Comment by seanbrookes** [comment]
test please
